demos/src/radios.mustache

```diff
@@ -11,12 +11,12 @@
 </div>
 <div>
 	<div class="o-forms-group o-forms--valid">
-		<label class="o-forms-label">Label (valid)</label>
```

Comment:
Note for review: valid states were exactly the same as… default states. So no need for them